### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by SanyuktaSaha**
added more tutorial pages to learn about the ladybug and its tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,23 @@ Ladybug imports standard EnergyPlus Weather files (.EPW) in Grasshopper and prov
...
+
+- More tutorials for Ladybug on (Performance + Form).
+
+"Performance+FORM is a platform to learn, SHARE, and collaborate on projects and WORKFLOWS for environmentally responsive design." The project is started by faculty and students at USC but is open to all. 
+
+### Other tutorials:
+
+- [A pdf tutorial](https://docs.google.com/file/d/0Bz2PwDvkjovJclJDMHJVMHJWaFU/edit) by Anna Kaertner and Chantal Jahn prepared for the [Building Technology Laboratory Class](http://architecture.mit.edu/subject/fall-2013-4411) at MIT taught by Les Norford. This tutorial goes through almost all of the Ladybug components except the sun-path.
+
+- ["Tutorial on the Use of Daysim Simulations for Sustainable Design"](https://www.dropbox.com/s/ril32ukfzgu2ci2/Daysim3.0.Tutorial.pdf) is written by Christoph F. Reinhart. From the document: " No matter what your level of expertise with daylight simulations is
```
Comment (chriswmackey) [inline README.md]: I don't think this is quite the right location for these resources. If you want to replace all of this with a link to the resources page on our website (https://www.ladybug.tools/resource.html), I think that would be more appropriate.

Comment (chriswmackey) [reply README.md]: @mostaphaRoudsari ,
Let me know what you think about this.

Comment (mostaphaRoudsari) [reply README.md]: I like the idea of replacing the resources with the link on ladybug.tools page. In that case we won't be concerned about keeping two places updated.

**Comment by mostaphaRoudsari** [comment]
Closing this because of inactivity.

**Pull request closed without merging**